src/sections/footer.html

```diff
@@ -1,4 +1,48 @@
-<div>
-    <h1>Footer</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
+<div class="lg:hidden text-darkgreen text-base md:text-base font-medium">
+    <div class="md:flex justify-between md:mb-5"> 
+        <div id="footer__address" class="mb-5">
+            <p>Agentur Baumeister</p> 
+            <p>Böckhstraße 13</p>
+            <p>10967 Berlin</p>
+        </div>
+        <div id="footer__contact" class="mb-28 md:mb-0 md:text-right">
+            <p class="underline">hallo@agentur-baumeister.com</p>
+            +49 (0)30 - 490 827 87
+        </div>
+    </div>
+   <ul
+    id="footer__links"
+    class="flex justify-between mb-7 md:mb-9"
+  >
+    <li>
+      <a href="#" target="_blank">Datenschutz</a>
+    </li>
+    <li>
+      <a href="#" target="_blank">Impressum</a>
+    </li>
+    <li>
+      <a href="#" target="_blank">LinkedIn</a>
+    </li>
+    <li>
+      <a href="#" target="_blank">Instagram</a>
+    </li>
+  </ul>
+</div>
+
+<div class="hidden lg:flex justify-between text-darkgreen text-[clamp(1rem, 1.58vw, 1.5rem)] font-medium">
```

Comment:
This shouldn't be green either.
```suggestion
<div class="hidden lg:flex justify-between text-[clamp(1rem, 1.58vw, 1.5rem)] font-medium">
```